onchain/rollups/test/foundry/dapp/Application.t.sol

```diff
@@ -978,8 +992,7 @@ contract ApplicationTest is TestBase {
                 inputIndexWithinEpoch: v.inputIndexWithinEpoch.toUint64(),
                 outputIndexWithinInput: v.outputIndexWithinInput.toUint64(),
                 outputHashesRootHash: v.outputHashesRootHash,
-                vouchersEpochRootHash: v.vouchersEpochRootHash,
-                noticesEpochRootHash: v.noticesEpochRootHash,
+                outputsEpochRootHash: v.noticesEpochRootHash,
```

Comment:
Once the off-chain component is ready, we should update this and encoding and the helper library `LibServerManager`

Reply:
Yes, many lines in this PR are mere artifacts that let us test the new output validation scheme using the old machine.